README.md

```diff
@@ -2,4 +2,28 @@ vs_scanwatch
 
 The function of this service is to scan the contents of a specified folder and attempt to move it's contents to a subfolder inside this folder. 
```

Comment:
It is a good practice to have Readme file split into logical sections, e.g. Overview, Configuration, Requirements, etc. 

For example, in the current README.md, it is really hard to see how can you do the configuration, because everything is mixed up. I would like to know if you can add environment variable that will override current (default) setting. I see that you mention keys, however, you need to be more precise where and how these keys are used.